src/_sass/wp.scss

```diff
@@ -1,37 +1,42 @@
-#projects .col1 {
+#projects-openoakland, #projects-active, #projects-inactive, #projects-decommissioned .col1 {
```

Comment:
For maintainability, it might be easier to instead create a `projects` class and apply it to these headers. 
[Kramdown classes](https://about.gitlab.com/blog/2016/07/19/markdown-kramdown-tips-and-tricks/#applying-classes)

Reply:
I was able to (mostly) resolve this without the need for classes, actually. Thanks for the nudge.